Dockerfile

```diff
@@ -5,6 +5,7 @@ WORKDIR /workspace
 # Copy the Go Modules manifests
 COPY go.mod go.mod
 COPY go.sum go.sum
+COPY local/ local/
```

Comment:
Why we need this?

Reply:
I was using local copy of your metallb changes for the test, hence this change.
now changed it to use `github.com/fedepaol/metallb v0.1.1-0.20220223172532-f7f3489fc501` in go.mod.